psydac/feec/multipatch/examples/mixed_source_pbms_conga_2d.py

```diff
@@ -66,15 +69,15 @@ def solve_magnetostatic_pbm(
 
       Gh: V0h -> V1h  and  Ah: V1h -> V1h
 
-    in a broken-FEEC approach involving a discrete sequence on a 2D multipatch domain \Omega,
+    in a broken-FEEC approach involving a discrete sequence on a 2D multipatch domain \\Omega,
 
       V0h  --grad->  V1h  -—curl-> V2h
 
     and boundary conditions to be specified (see the multi-patch paper for details).
 
     Harmonic constraint: if dim_harmonic_space > 0, a constraint is added, of the form
 
-        u in H^\perp
+        u in H^\\perp
```

Comment:
the double backslash is there for a latex rendering of the docstring ? (just out of curiosity)

Reply:
I was using autopep8 to get a consistent formatting across all files. Apparently it escapes '\' in strings, we can discuss if this is bad or not.